tests/components/androidtv_remote/test_media_player.py

```diff
@@ -0,0 +1,344 @@
+        "media_player",
+        "volume_set",
+        {"entity_id": MEDIA_PLAYER_ENTITY, "volume_level": 0.13},
+        blocking=True,
+    )
+
+    mock_api.send_key_command.assert_called_with("VOLUME_DOWN", "SHORT")
+    assert mock_api.send_key_command.call_count == 2
+
+
+async def test_media_player_controls(
+    hass: HomeAssistant, mock_config_entry: MockConfigEntry, mock_api: MagicMock
+) -> None:
+    """Test the Android TV Remote media player play/pause/stop/next/prev."""
+    mock_config_entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    assert mock_config_entry.state is ConfigEntryState.LOADED
+
+    assert await hass.services.async_call(
+        "media_player",
+        "media_play",
+        {"entity_id": MEDIA_PLAYER_ENTITY},
+        blocking=True,
+    )
+
+    mock_api.send_key_command.assert_called_with("MEDIA_PLAY", "SHORT")
+
+    assert await hass.services.async_call(
+        "media_player",
+        "media_pause",
+        {"entity_id": MEDIA_PLAYER_ENTITY},
+        blocking=True,
+    )
+
+    mock_api.send_key_command.assert_called_with("MEDIA_PAUSE", "SHORT")
+
+    assert await hass.services.async_call(
+        "media_player",
+        "media_play_pause",
+        {"entity_id": MEDIA_PLAYER_ENTITY},
+        blocking=True,
+    )
+
+    mock_api.send_key_command.assert_called_with("MEDIA_PLAY_PAUSE", "SHORT")
+
+    assert await hass.services.async_call(
+        "media_player",
+        "media_stop",
+        {"entity_id": MEDIA_PLAYER_ENTITY},
+        blocking=True,
+    )
+
+    mock_api.send_key_command.assert_called_with("MEDIA_STOP", "SHORT")
+
+    assert await hass.services.async_call(
+        "media_player",
+        "media_previous_track",
+        {"entity_id": MEDIA_PLAYER_ENTITY},
+        blocking=True,
+    )
+
+    mock_api.send_key_command.assert_called_with("MEDIA_PREVIOUS", "SHORT")
+
+    assert await hass.services.async_call(
+        "media_player",
+        "media_next_track",
+        {"entity_id": MEDIA_PLAYER_ENTITY},
+        blocking=True,
+    )
+
+    mock_api.send_key_command.assert_called_with("MEDIA_NEXT", "SHORT")
+
+
+async def test_media_player_play_media(
+    hass: HomeAssistant, mock_config_entry: MockConfigEntry, mock_api: MagicMock
+) -> None:
+    """Test the Android TV Remote media player play_media."""
+    mock_config_entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    assert mock_config_entry.state is ConfigEntryState.LOADED
+
+    assert await hass.services.async_call(
+        "media_player",
+        "play_media",
+        {
+            "entity_id": MEDIA_PLAYER_ENTITY,
+            "media_content_type": "channel",
+            "media_content_id": "45",
+        },
+        blocking=True,
+    )
+    assert mock_api.send_key_command.mock_calls == [
+        call("4", "SHORT"),
+        call("5", "SHORT"),
+    ]
+
+    assert await hass.services.async_call(
```

Comment:
Why are you calling media_player.play_media twice for channel 45? Add a comment if that was intentional.